activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1185,11 +1185,21 @@ def options_include_default?(options)
       end
 
       # Changes the comment for a table or removes it if +nil+.
+      #
+      # Passing a hash containing +:from+ and +:to+ will make this change
+      # reversible in migration:
+      #
+      #   change_table_comment(:posts, from: "old_comment", to: "new_comment")
       def change_table_comment(table_name, comment)
```

Comment:
How about changing `comment` to `comment_or_changes` like `change_column_default`?

```suggestion
      def change_table_comment(table_name, comment_or_changes)
```

https://github.com/rails/rails/blob/877376eba155969d66fc243306ea4a9bd9abb54a/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L640

Reply:
Thanks. I'll push this later to avoid frequent push.